crates/gpui/src/platform/windows/window.rs

```diff
@@ -455,9 +455,13 @@ impl WindowsWindowInner {
         if first_char.is_control() {
             None
         } else {
+            let key = match first_char {
+                ' ' => "space".to_string(),
+                first_char => first_char.to_lowercase().to_string(),
+            };
```

Comment:
I don't believe any other keys need to be handled here. `WM_CHAR` doesn't report '\t' or '\n' as far as I could tell.